src/Hazelcast.Net/HazelcastOptions.cs

```diff
@@ -66,6 +67,10 @@ private HazelcastOptions(HazelcastOptions other)
             NearCaches = other.NearCaches.Select(kvp
                 => new KeyValuePair<string, NearCacheOptions>(kvp.Key, kvp.Value.Clone()))
                 .ToDictionary();
+
+            FlakeIdGenerators = other.FlakeIdGenerators.Select(kvp
```

Comment:
`other.FlakeIdGenerator.ToDictionary(x => x, x => x.Clone())` would work too?

Reply:
Just copied implementation above, should I also update it for `NearCaches`?

Reply:
sure